txnkv/transaction/pipelined_flush.go

```diff
@@ -481,7 +480,7 @@ func (c *twoPhaseCommitter) resolveFlushedLocks(bo *retry.Backoffer, start, end
 		fmt.Sprintf("pipelined-dml-%s", status),
 		fmt.Sprintf("pipelined-dml-%s-%d", status, c.startTS),
 		c.store,
-		RESOLVE_CONCURRENCY,
+		c.txn.pipelinedResolveLockConcurrency,
```

Comment:
The throttle flow control works when flushing.

And the resolve-lock phase can also consume many resources, and it may also needs to be controlled by throttle, what about adding a TODO here.

Reply:
Throttling the resolve lock goroutine is in our current plan I think. For now we rely on the concurrency to constrain it. If we want a better control of its resource usage we might need to consider a more comprehensive mechanism, e.g. one that helps reduce the overhead of resolving locks of multiple p-txns running in parallel.